docs/drafts/badges.rst

```diff
@@ -0,0 +1,144 @@
+Badges
+======
+
+This is a summary of the "badge concept" user story. The concepts are
+not yet implemented in Adhocracy 3.
+
+Adhocracy 2 knows a variety of badges and similar entities. The aim is
+to streamline these entities into more consistent or possibly independent
+concepts in Adhocracy 3. Some aspects shall not be implemented in A3.
+
+This document can be replaced once the concepts are ready in A3.
+
+
+Badge features in Adhocracy 2
+-----------------------------
+
+The various badges, categories and tags can be categorized as the
+following:
+
+
+Allowed targets
+    What can be badged?
+
+
+Locality
+    Do badges exist globally or locally only?
+
+Exclusivity
+    Can only one badge out of a certain group be assigned to a target
+    object?
+
+Hierarchy
+    Can badges be structured hierarchically?
+
+Dedicated pages
+    Does this badge have a dedicated page?
+
+
+Create permission
+    Who may create badges / choose available badges in a given context?
+
+Assign permission
+    Who may assign badges to targets?
+
+View permission
+    Who may view badges?
+
+
+Color
+    If shown as normal badge, what color should it have?
+
+Icon
+    If shown as thumbnail, which icon should be shown?
+
+Visibility
+    Should it be shown at all in listings?
+
+
+Impact
+    Effect on mixed list sortings
+
+Implicit user role
+    If a user has this badged, which additional role should she have?
+    (we should drop this)
+
+Voteable
+    Allow users to vote whether the assignment applies (tags in
+    Adhocracy 1)
+
+Behaviour
+    Assign a certain behaviour to a badge (research project at HHU)
+
+
+Badges, categories and tags in A2
+---------------------------------
+
+Some remarkable aspects of badge-like entities in A2:
+
+
+Default
```

Comment:
What dou you mean by default? Is that the default behaviour of badges, categories and tags? Or is this some abstrct class, that all badge-like entities in A2 inherit?